romanesco/format/__init__.py

```diff
@@ -54,17 +59,97 @@ def csv_to_rows(input):
 
     return output
 
-converters = {}
-validators = {}
+
+def converter_path(source, target):
+    """Gives the shortest path that should be taken to go from a source
+    type/format to a target type/format.
+
+    Throws a NetworkXNoPath exception if it can not find a path.
+
+    Returns a list of edges in the order they should be traversed.
+    """
+    # These are to ensure an exception gets thrown if source/target don't exist
+    get_validator(*source)
```

Comment:
We should make the specification of source and target consistent between the functions so we don't need to use `*`. I'm fine with your namedtuple suggestion, but if we go with that, let's make all of the functions expect that type, and we should document it with sphinx conventions for said functions.